spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -94,9 +96,7 @@ public class TestRewritePositionDeleteFilesAction extends SparkCatalogTestBase {
   private static final int SCALE = 4000;
   private static final int DELETES_SCALE = 1000;
 
-  @Parameterized.Parameters(
-      name =
-          "formatVersion = {0}, catalogName = {1}, implementation = {2}, config = {3}, fileFormat = {4}")
+  @Parameters(name = "catalogName = {0}, implementation = {1}, config = {2}, fileFormat = {3}")
```

Comment:
this should also include `formatVersion`, right?

Reply:
`formatVersion` is not provided, nor is there a variable with that name in the class. It was only added in the Parameters() name. So I removed it from the name. 

I think when this file was made from `TestPositionDeletesTable`, they simply copied the Parameters() as it is. If you see `TestPositionDeletesTable`, it also exists only in the Parameters() name and no `format` variable exists in the class.